tgui/packages/tgui/hotkeys.ts

```diff
@@ -135,9 +139,9 @@ export const releaseHeldKeys = () => {
 
 export const setupHotKeys = () => {
   // Read macros
-  Byond.winget('default.*').then(data => {
+  Byond.winget('default.*').then((data: any) => {
```

Comment:
Why? This is a type we control. Should probably at least be a `Record` of some kind since we're using Object.keys.

From the code, it looks like:

```ts
type ByondHotkey = {
    command: string;
    name: string;
}

data: Record<string, ByondHotkey>
```